pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -30,12 +32,8 @@ export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> =
     TEMPORARY_DOMAINS.find((domain) => {
       return domain.title === portfolio.domain;
     });
-  const component =
-    portfolio &&
-    TEMPORARY_COMPONENTS.find((component) => {
-      return component.id === "aa78c93b-d57d-4d9a-b255-e8cfe26bde48";
-    });
-  const badgeNumber = 1;
+  const components = TEMPORARY_COMPONENTS;
+  const badgeNumber = components.results.length;
   const badgeLabel = badgeNumber < 100 ? _.toString(badgeNumber) : "99+";
   const maxItems = badgeNumber > 100;
```

Comment:
I think `maxItems` should already be set if it's 100, not only if it's more than 100.

So: `>= 100`.

Reply:
I'd opt to just remove this variable.